app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -44,16 +47,15 @@ trait UserRoutes extends Authentication with PaginationDirectives with UserError
     }
   }
 
-  def listUsers: Route = authenticate { user =>
+  def listUsers: Route = authenticateRootMember { user =>
```

Comment:
I think we should leave this until we figure out uses for the `Admin` rule and intra-organization permissions.